### PR TITLE
Feature/use mailservers

### PR DESCRIPTION
**Pull request by cammellos**
Add a use-mailservers flag to status-go

### CODE REVIEW

File: waku/waku.go [hunk 1]
```diff
@@ -1083,7 +1084,14 @@ func (w *Waku) HandlePeer(peer common.Peer, rw p2p.MsgReadWriter) error {
...
 
-	w.logger.Debug("handling peer", zap.String("id", types.EncodeHex(peer.ID())))
+	peerID := types.EncodeHex(peer.ID())
+	w.logger.Info("handling peer", zap.String("id", peerID), zap.String("ip", peer.IP().String()))
+
+	/*
+		if w.softBlacklisted(peerID) || "36.148.71.37" == peer.IP().String() || "47.244.23.27" == peer.IP().String() {
+			w.logger.Info("peer is blacklisted, disconnecting", zap.String("peer", peerID), zap.String("ip", peer.IP().String()))
+			return errors.New("blacklisted")
+		}*/
```
Comment (Samyoul) [inline waku/waku.go]: This should be removed.

File: waku/v0/peer.go [hunk 2]
```diff
@@ -173,6 +173,10 @@ func (p *Peer) EnodeID() enode.ID {
 	return p.p2pPeer.ID()
 }
 
+func (p *Peer) Name() string {
+	return p.p2pPeer.Name()
+}
+
```
Comment (Samyoul) [inline waku/v0/peer.go]: 🙂 Updating a deprecated API to ensure compatibility with the Peer interface.

I don't have a better solution.

**Review by Samyoul — APPROVED** [review]
Apart from the commented code that slipped in, all good.

**Comment by cammellos** [comment]
Oh thanks @Samyoul, I completely missed the extra code

**Pull request merged**